conf/config.json.example

```diff
@@ -1,13 +1,12 @@
-
 {
     "production": {
         "protocolUseSSL": true,
         "domain": "__DOMAIN__",
-        "port": __PORT__,
+        "port": "__PORT__",
         "loglevel": "info",
         "useCDN": false,
         "allowGravatar": false,
-        "allowPDFExport": true,
+        "allowFreeURL": true,
```

Comment:
Is it intended to actually use free-URL-mode, while it provides a etherpad-like experience, I wouldn't recommend it, as it sometimes causes issues with unintended pathnames that are used, for example, by bots that end up as notes (e.g. `wp-login.php` suddenly becomes a note because some odd bot thought it would be great to scan for wordpress)

Reply:
Thanks very much for the review!

Reply:
@ericgaspar and @lapineige do we switch to `"allowFreeURL": false,` ?

Reply:
@yalh76  I would put `allowFreeURL` to `false` for now. This option could be documented and activated as an option by users.

Reply:
`allowFreeURL `disbaled